packages/machines/editable/src/editable.machine.ts

```diff
@@ -28,6 +29,10 @@ export function machine(ctx: UserDefinedContext) {
         },
       },
 
+      watch: {
+        value: ["invokeOnChange", "setInputValue"],
```

Comment:
```suggestion
        value: ["invokeOnChange", "syncInputValue"],
```

Let's use a better name for that, so it's not confusing. There's setValue, and then there's setInputValue.